app/riot/src/client/preview/render.js

```diff
@@ -0,0 +1,23 @@
+import { document } from 'global';
+import { stripIndents } from 'common-tags';
+import { mount, unregister, tag2 } from 'riot';
+import compiler from 'riot-compiler';
+import { render as renderRiot } from './render-riot';
+
+const rootElement = document.getElementById('root');
+
+export default function renderMain({ story, selectedKind, selectedStory, showMain, showError }) {
+  showMain();
+  rootElement.innerHTML = '<root></root>';
+  const context = { unregister, mount, tag2, compiler, rootElement };
```

Comment:
Looks like `unregister, mount, tag2, compiler` are just riot imports, why do you need them as a context?

Reply:
because of the jest tests failing with babel not able to recognize the import notation.

Maybe you can give me a hand to be able to run the jest tests without having to inject the riot import inside the context.

Reply:
Do you have an error log?